File: VB/WindowsApplication133/Form2.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.XtraPivotGrid
Imports DevExpress.Utils
Imports DevExpress.XtraEditors
Imports DevExpress.XtraEditors.Controls
Imports DevExpress.XtraPivotGrid.Customization
Imports System.Collections
Imports System.Xml.Serialization
Imports System.IO
Imports DevExpress.XtraPivotGrid.Data

Namespace E4592
	Partial Public Class Form2
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub Form2_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
			' TODO: This line of code loads data into the 'nwindDataSet.Invoices' table. You can move, or remove it, as needed.
			Me.invoicesTableAdapter.Fill(Me.nwindDataSet.Invoices)
			Dim helper As New CustomizationFindHelper(pivotGridControl1)
			pivotGridControl1.FieldsCustomization()
		End Sub

	End Class
End Namespace

File: CS/WindowsApplication133/Form2.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraPivotGrid;
using DevExpress.Utils;
using DevExpress.XtraEditors;
using DevExpress.XtraEditors.Controls;
using DevExpress.XtraPivotGrid.Customization;
using System.Collections;
using System.Xml.Serialization;
using System.IO;
using DevExpress.XtraPivotGrid.Data;

namespace E4592
{
    public partial class Form2 : Form
    {
        public Form2()
        {
            InitializeComponent();
        }

        private void Form2_Load(object sender, EventArgs e)
        {
            // TODO: This line of code loads data into the 'nwindDataSet.Invoices' table. You can move, or remove it, as needed.
            this.invoicesTableAdapter.Fill(this.nwindDataSet.Invoices);
            CustomizationFindHelper helper = new CustomizationFindHelper(pivotGridControl1);
            pivotGridControl1.FieldsCustomization();
        }

    }
}